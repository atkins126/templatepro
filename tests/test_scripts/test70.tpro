{{include("included1.tpro")}}
{{:value2|sayhello}}
{{include("partials/included2.tpro")}}
{{:value2|sayhello}}
{{include("included1.tpro")}}
{{:value2}} => {{:value2|lowercase}}
{{include("included1.tpro")}}