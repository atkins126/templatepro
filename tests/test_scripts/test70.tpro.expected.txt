I'm from the included file
This is value 1 true
Hello There
I'm from the included file in partials folder
This is value 1 true
Hello There
I'm from the included file
This is value 1 true
DANIELE2 => daniele2
I'm from the included file
This is value 1 true